README.rst

```diff
@@ -12,10 +12,20 @@ at https://github.com/enovance/edeploy.
 
 eDeploy-roles is a set of tools to build roles to be used with eDeploy.
 
+Requirement
+-----------
+
+You need to get the ``edeploy`` repository and generate the ``base`` role first::
+
+  $ git clone http://github.com/enovance/edploy
+  $ sudo ln -s $PWD/edeploy /srv/edeploy
```

Comment:
Yes, too. That's just how I install it on my computer. This way, I keep the git repository in my home directory.
Should I adjust the patch to install it directly in /srv/edeploy?